CHANGES.md

```diff
@@ -1,3 +1,33 @@
+1.7.2 / 2019-09-XX
+==================
+
+Bug fixes:
+
+* DriverInterface#getValue() can return null
+* Modify response[Contains|NotContains] to use strpos vs preg_match
+* Fix compatibility with PHP 7.2
+* Allow Symfony 4.
+* Optimize the splitting of XPath unions
+* Allow pipes (|) in attributes by implementing better XPath parsing
+* Auto-start session only upon 1st "->visit(...)" method call
+
+Testsuite:
+
+* Don't install PHPUnit 8.0
+* Fix PHP 5.4 and PHP 5.5 builds
+* Objects with __toString() can be asserted.
+* Add testing on PHP 7.2
+* Add compatibility with PHPUnit 6
+* Extracted driver test suite into separate repository.
+
+Misc:
+
+* Rename boolean type in lowercase
```

Comment:
Please remove this line.

---

This is an internal change, that shouldn't be affecting developer workflow.

Reply:
this one should be listed IMO, as it precisely help developers running Psalm on a project using Mink (that was the reason for the change)

Reply:
Isn't that the case, where `vendor` folder was analyzed by mistake 😄 . Let's leave it then.

Reply:
to analyze your own code *using Mink*, psalm needs to know the types of Mink APIs. And so it will look at the phpdoc (and it seems it considers `Boolean` as a class name)